dashboard/test/ui/runner.rb

```diff
@@ -23,14 +23,16 @@
 require 'colorize'
 require 'open3'
 require 'parallel'
+require 'socket'
 
 require 'active_support/core_ext/object/blank'
 
 ENV['BUILD'] = `git rev-parse --short HEAD`
 
+GIT_BRANCH = GitUtils.current_branch
 COMMIT_HASH = RakeUtils.git_revision
 S3_LOGS_BUCKET = 'cucumber-logs'
-S3_LOGS_PREFIX = GitUtils.current_branch
+S3_LOGS_PREFIX = "#{Socket.gethostname}/#{GIT_BRANCH}"
```

Comment:
Noticing on Circle machines this ends up being e.g. `box6`, should be fine for now. A few other options while I'm thinking about it:

* `ENV['CI'] ? 'circle' : Socket.gethostname` - might be clearest when looking at root directory on S3
* `ENV['CI'] ? ENV['CIRCLE_BUILD_NUM'] : Socket.gethostname` - would support simultaneous runs on the same branch (e.g. if flakiness-testing a change by re-running a bunch at once as I found useful over the weekend)